visualization/dashboard/src/Localization/en.json

```diff
@@ -255,7 +255,9 @@
         "datasetCohortSelector": "Select a dataset cohort",
         "_datasetCohortSelector.comment": "label for selecting single cohort",
         "aggregateFeatureImportance": "Aggregate Feature Importance",
-        "_aggregateFeatureImportance.comment": "graph label for aggregated (summed and averaged) feature importances"
+        "_aggregateFeatureImportance.comment": "graph label for aggregated (summed and averaged) feature importances",
+        "missingParameters": "The aggregate feture importance tab requires the local feature importance parameter be supplied.",
```

Comment:
@mesameki These are the complete list of strings for the warnings. All names that start with _ and end with .comment are to help translators, not presented in the actual dashboard.